docs/user/dashboard/tutorial-create-a-dashboard-of-lens-panels.asciidoc

```diff
@@ -66,296 +97,193 @@ image::images/lens_end_to_end_1_3.png[Changed type and dropped clientip field]
+. In the toolbar next to *Line* click the *Left axis* icon, then deselect *Show*.
 +
 [role="screenshot"]
-image::images/lens_end_to_end_3_3.png[Change layer type]
-+
-The chart type for the visualization changes to *Mixed XY*.
-
-. From the *Available fields* list, drag and drop the data fields to the *Drop a field or click to add* fields in the editor.
-
-.. Drag and drop *timestamp* to the *Horizontal axis* field.
-
-.. Drag and drop *clientip* to the *Vertical axis* field.
-
-. Change the *timestamp* interval.
+image::images/lens_end_to_end_4_3.png[Turn off axis name]
 
-.. In the editor, click *timestamp* in the line chart layer.
+. Click the *Bottom axis* icon, then deselect *Show*.
 
-.. Select *Customize time interval*.
+. Click *Save and return*
 
-.. Change the *Minimum interval* to `1 days`, then click *Close*.
+. Arrange the panel so that it is in the same row as the *Metric* visualization. The two should
+take up half the screen width.
 
-. Change the *Unique count of clientip* label and color.
+. Add a panel title to explain the visualization, which is necessary because you removed the axis labels.
 
-.. In the editor, click *Unique count of clientip*.
+.. Click the gear icon and choose *Edit panel title*.
 
-.. In the *Display name* field, enter `Unique visitors` in the line chart layer.
+.. Enter "Median of bytes" as the panel title, then click *Save*.
 
-.. In the *Series color* field, enter *#CA8EAE*, then click *Close*.
+. Click *Save* on the entire dashboard.
 
 [discrete]
-[[configure-the-multiaxis-chart]]
-==== Configure the y-axes
-
-There is a significant difference between the *timestamp per day* and *Unique visitors* data, which makes the *Unique visitors* data difficult to read. To improve the readability, 
-display the *Unique visitors* data along a second y-axis, then change the formatting. When functions contain multiple formats, separate axes are created by default. 
-
-. In the editor, click *Unique visitors* in the line chart layer.
+[[view-the-distribution-of-visitors-by-operating-system]]
+=== View the most frequent values
 
-. For *Axis side*, click *Right*, then click *Close*.
+When you drag a text or IP address field into the *Lens* workspace you will see
+the most frequent values of that field using the *top values* function. The values
+are ranked based on another metric like *Count of records*. Only exact matches count:
+`win 10` and `win xp` are distinct.
 
-[float]
-[[change-the-visualization-type]]
-==== Change the visualization type
+For this tutorial, you have decided to see the most frequently viewed URLs on your website
+ranked by the `Unique count of clientip` metric, but you have not yet determined which chart type to use.
 
-. In the editor, click *Average of bytes* in the area chart layer. 
+. Starting with the default bar chart, drag the `clientip` field onto the *Vertical axis*.
+This will produce the `Unique count of clientip` metric. Do not drop the field into the main workspace,
```

Comment:
There are a lot of instances of `will` in this tutorial. `Will` implies that this is a future action, but if users are following along, these actions are happening in real time. It is best to change all of these to active voice. For example:

`*Lens* applies the *Unique count* function. Do not drag and drop `clientip` onto the visualization builder because the *Top values* function is applied instead.